pkg/endpoints/credentials.go

```diff
@@ -153,6 +156,20 @@ func (r Resource) createCredential(request *restful.Request, response *restful.R
 		return
 	}
 
+	// Patch service account
+	saName := ""
+	providedSaName := os.Getenv("PIPELINE_RUN_SERVICE_ACCOUNT")
+	if providedSaName != "" {
+		saName = providedSaName
```

Comment:
PIPELINE_RUN_SERVICE_ACCOUNT isn't anywhere in the code atm (e.g. in https://github.com/tektoncd/dashboard/blob/master/config/tekton-dashboard-deployment.yaml), so actually it's worth doing the following

1) omit this altogether and document in the readme the dashboard must use the default service account
2) add the variable into our deployment yaml and test all is good with and without the service account being set

Reply:
I did the "2".